assets/shaders/chunk_frag.glsl

```diff
@@ -174,6 +175,23 @@ void main() {
             discard;
         }
     #endif
+
+    /* APPLY OVERALL BIOME COLOR OFFSET */
+    if (v_blockHint != BLOCK_HINT_GRASS) {
```

Comment:
Does grass have a hard-coded mask? Not sure we ever had that, but I think being able to provide an override-able texture as mask for generic blocks would be great. Would it be hard to support that?

Reply:
umm not sure you can do anything about this though?

Reply:
No idea how feasible that is, but right now it's just hardcoded in the engine. See the right side of this texture: https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/resources/org/terasology/engine/assets/textures/effects.png

Reply:
good part: we already take the mask out of a texture file 😃 

bad part: it is hard-coded for a very specific block 🙈 

This would probably be worth a follow-up later on to see whether we can (a) at least override the `effects.png` texture to match the mask with different grass textures (e.g., in Light and Shadow), and (b) specify arbitrary effect masks in the block definition.